backend/common/db/models/__init__.py

```
@@ -30,6 +30,25 @@
     CohortInvite,
 )
 
+# Simulation runtime models
+from .simulation import (
+    UserProgress,
+    StudentSimulationInstance,
```

Ruff (0.14.8)

Redefinition of unused `StudentSimulationInstance` from line 27

(F811)